bin/automated

```diff
@@ -35,45 +33,6 @@ function doBuild() {
         cd $ROOT_DIR
         echo "$talk generated"
     done
-
-    # Delete and move files.
-    cp CNAME _site
-    find . -maxdepth 1 ! -name '_site' ! -name '.git' ! -name '.gitignore' -exec rm -rf {} \;
-    mv _site/* .
-    rm -R _site/
-    rm Gemfile
-    rm Gemfile.lock
 }
```

Comment:
Since this script does nothing else than a build now, is it necessary to wrap all these commands in a function?